app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSerializer < ActiveModel::Serializer
```

Comment:
The other actions can serialize using them by specifying it directly in the render. I believe there may also be a config that can help with this but I'd have to go look at the AMS docs.